node/src/mpc_client.rs

```diff
@@ -312,14 +191,136 @@ impl MpcClient {
             ),
         );
 
-        monitor_passive_channels.await?;
+        let _ = monitor_passive_channels.await?;
         monitor_chain.await?;
         generate_triples.await??;
         generate_presignatures.await??;
 
         Ok(())
     }
 
+    async fn monitor_passive_channels_inner<T>(
+        mut channel_receiver: mpsc::Receiver<NetworkTaskChannel>,
+        mpc_client: MpcClient,
+        auto_abort: tracking::AutoAbortTask<T>,
+    ) -> anyhow::Result<(), JoinError> {
+        let _auto_abort_metrics = auto_abort;
```

Comment:
wait I'm not sure why _auto_abort_metrics needs to be in here - we can just have the run() function keep that as a local variable. Don't need to move it in here.

Reply:
`ping` @robin-near.